src/Datagrid/Pager.php

```diff
@@ -69,7 +69,7 @@ abstract class Pager implements \Iterator, \Countable, \Serializable, PagerInter
 
     // used by iterator interface
     /**
-     * @var \Traversable|array|null
+     * @var object[]&\Traversable|null
```

Comment:
Why Traversable ?
```
    /**
     * Returns an array of results on the given page.
     *
     * @return array
     */
    public function getResults();
```

Reply:
Don't know. It was added 4 years ago: https://github.com/sonata-project/SonataAdminBundle/pull/4265/files#diff-bad535da966d43dbce66a002d626f35cR70

Reply:
I would try to remove it, since it's either null, either the return value of `getResults`